docs/developers/index.md

```diff
@@ -149,6 +149,12 @@ Ethereum has a large and growing number of tools to help developers build, test,
 
 ## Integrated Development Environments (IDEs)
 
+**Ethereum Studio -** ***Official Ethereum on-boarding IDE. Web-based IDE with a built in browser blockchain, MetaMask integration, transaction logger and many other features to help you getting started coding DApps as fast as possible. ***
```

Comment:
Remove the space before the final "***" for proper markdown formatting. Current display:
<img width="836" alt="Image 2019-10-13 at 8 20 46 PM" src="https://user-images.githubusercontent.com/8097623/66714885-a906a200-edf7-11e9-8efb-9c084e5d027a.png">